app/src/main/java/com/ucmobiledevelopment/freeways/MainActivity.kt

```diff
@@ -30,7 +30,7 @@ class MainActivity : ComponentActivity() {
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colors.background
                 ) {
-                    Greeting("Android")
+                    greeting("Android")
```

Comment:
Changed function name to lowercase because it is a single word.

Reply:
I really want to agree with you on this, as it is normally the case, but composable functions should be nouns, and PascalCase, believe it or not.  
https://developer.android.com/kotlin/style-guide